stl/inc/__msvc_chrono.hpp

```diff
@@ -720,62 +720,62 @@ _NODISCARD bool _To_xtime_10_day_clamped(_CSTD xtime& _Xt, const _CHRONO duratio
 
 inline namespace literals {
     inline namespace chrono_literals {
-        _EXPORT_STD _NODISCARD constexpr _CHRONO hours operator"" h(unsigned long long _Val) noexcept
+        _EXPORT_STD _NODISCARD constexpr _CHRONO hours operator""h(unsigned long long _Val) noexcept
         /* strengthened */ {
             return _CHRONO hours(_Val);
         }
 
-        _EXPORT_STD _NODISCARD constexpr _CHRONO duration<double, ratio<3600>> operator"" h(long double _Val) noexcept
+        _EXPORT_STD _NODISCARD constexpr _CHRONO duration<double, ratio<3600>> operator""h(long double _Val) noexcept
         /* strengthened */ {
             return _CHRONO duration<double, ratio<3600>>(_Val);
         }
 
-        _EXPORT_STD _NODISCARD constexpr _CHRONO minutes(operator"" min)(unsigned long long _Val) noexcept
+        _EXPORT_STD _NODISCARD constexpr _CHRONO minutes(operator""min)(unsigned long long _Val) noexcept

```

Comment:
IIUC the parentheses around `operator""min` was used to defend against macroized `min`. Since `min` is now a part of the `""min` token, are they still needed?

```suggestion
        _EXPORT_STD _NODISCARD constexpr _CHRONO minutes operator""min(unsigned long long _Val) noexcept
```

Reply:
I'm surprised that the preprocessor won't expand the macro here (and verified so).